src/dlfcn.c

```diff
@@ -186,13 +186,20 @@ static void save_err_str( const char *str )
 
 static void save_err_ptr_str( const void *ptr )
 {
-    char ptr_buf[19]; /* 0x<pointer> up to 64 bits. */
+    char ptr_buf[2 + 2 * sizeof( ptr ) + 1];

```

Comment:
I am a bit dubious about this, as VLA are not part of C++, but perhaps given that `sizeof(ptr)` is know at compile time this is not a VLA?

Reply:
That is a good question. According to C standards it is not VLA because sizeof is evaulated at compile time. But how it behaves in MSVC is another question. We can wait for AppVeyor testing and will see... But if there is a problem that sizeof in array size can be replaced by number 8.

Reply:
Ok, so tests passed, warning disappeared and there is no problem with sizeof.